### ISSUE TITLE
start.sh (Docker Quickstart Terminal) in windows fails to start - workaround

### ISSUE THREAD

**Issue by sanori** [docker/machine#1786]
It occurs Window DockerToolbox-1.8.1b.exe with git-scm 2.5.1 with mintty.

After several VM install instructions, it stops with following messages
```
Machine default already exists in VirtualBox.
Starting machine default...
Started machines may have new IP addresses. You may need to re-run the `docker-machine env` command.
Setting environment variables for machine default...
Error: Unknown shell
bash: On: command not found
Looks like something went wrong... Press any key to continue...
```

After some investigation I found that `eval "$($DOCKER_MACHINE env $VM)"` instruction fails to set important environment variables. `docker-machine.exe env default` in bash does not work properly.
```
$ docker-machine.exe env default
On Windows, please specify either 'cmd' or 'powershell' with the --shell flag.


Error: Unknown shell
```

Different from the messages,  `docker-machine env --shell bash` makes environment variables  syntax for bash.

**Workaround:** I changed a line in `start.sh`
```
eval "$($DOCKER_MACHINE env $VM)"
```
to
```
eval "$($DOCKER_MACHINE env --shell bash $VM)"
```
and it works. But is this proper fix? Can `docker-machine env` detect the shell and output proper environment variable setting?

**Comment by qzole** [comment]
This was fixed in 1.8.1c, the solution is the same as yours:
https://github.com/docker/toolbox/commit/97968b663902523898f902bc77b953d0a98f3f21#diff-5ff432e42ca5c0757b31b4fe6f73f245

Also start.sh is not part of docker-machine, but of docker toolbox

**Comment by sanori** [comment]
Thanks!
I confirmed that it works well in 1.8.1c.
